Source/WebKit/UIProcess/wpe/WebPasteboardProxyWPE.cpp

```diff
@@ -292,6 +292,21 @@ void WebPasteboardProxy::getPasteboardItemsCount(IPC::Connection&, const String&
 
 void WebPasteboardProxy::readURLFromPasteboard(IPC::Connection& connection, size_t index, const String&, std::optional<WebPageProxyIdentifier>, CompletionHandler<void(String&& url, String&& title)>&& completionHandler)
 {
+#if ENABLE(WPE_PLATFORM)
+    if (usingWPEPlatformAPI()) {
+        if (index) {
```

Comment:
Can you document what does it mean that index is not zero?

Reply:
This is because we don't support multiple items in the clipboard